presto-orc/src/main/java/com/facebook/presto/orc/metadata/statistics/ColumnStatistics.java

```diff
@@ -57,6 +57,46 @@ public long getNumberOfValues()
         return numberOfValues == null ? 0 : numberOfValues;
     }
 
+    public boolean hasMinAverageValueSizeInBytes()
+    {
+        return hasNumberOfValues() && numberOfValues > 0;
+    }
+
+    /**
+     * The minimum average value sizes.
+     * The actual average value size is no less than the return value.
+     * It provides a lower bound of the size of data to be loaded
+     */
+    public long getMinAverageValueSizeInBytes()
```

Comment:
I think we should move this calculation to the caller because the ORC and DWRF readers have many cases where they "drop" stats for various reasons.  This would allow the caller to still calculate the size based on the column type, and then actually null out the stat.